scripts/partner_batch_imports.py

```diff
@@ -67,7 +67,9 @@
         'version',
         # Not a book
         'calendar',
+        'copy bin',
```

Comment:
Oh this might not work cause of the space; I believe it splits on space when it does the check. Also this filter only applies to independently published books

Reply:
Ah, ok, well those two that I added are likely to do nothing then as the dumpbins are from recognised publishers. It's not worth adding another title filter, the `OTH` check I added is doing the real work of preventing non-book items. The two strings can probably be removed to avoid confusion, although I don't think they will cause any false positives?

Looks like `copy bin` will never match because of the space. The `title_words` are split on space.

Reply:
Removing makes sense to me, but I'd also suggest creating an issue to expand this to block those words from any publisher source. (Potentially also worthwhile to switch this to regex so the space thing isn't a problem)